### PR TITLE
General bug fixes

### PR DESCRIPTION
**Pull request by jason-ng-zq99**
- Fix invalid function bug stemming from `Suspense` in `routes.tsx`

- Change all instances of `styled.text` to `styled.p`

- Add `margin: 0` to all new `styled.p` divs that do not have any `margin` attributes  

- Rename icon/logo pathnames from `%PUBLIC_URL%/icon` to `%PUBLIC_URL%/logo/icon`

### CODE REVIEW

File: frontend/src/components/Scheduling/ViewEventDetailCard.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const RowDescription = styled.div`
   justify-content: flex-start;
 `
 
-const StyledTitle = styled.text`
+const StyledTitle = styled.p`
```
Comment (hui444) [inline frontend/src/components/Scheduling/ViewEventDetailCard.tsx:69]: this needs `margin: 0 20px 0 0;`

File: frontend/src/routes/FacilityBooking/CreateBooking/oldindex.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ const StyledDateInput = styled(Input)`
     color: #d9d9d9;
   }
 `
-const StyledTitle = styled.text`
+const StyledTitle = styled.p`
```
Comment (hui444) [inline frontend/src/routes/FacilityBooking/CreateBooking/oldindex.tsx:67]: this requires `margin: 0 20px 0 0;`

File: frontend/src/routes/Schedule/CreateEvent/index.tsx [hunk 3]
```diff
@@ -51,7 +51,7 @@ const Row = styled.div`
   justify-content: space-between;
 `
 
-const StyledTitle = styled.text`
+const StyledTitle = styled.p`
```
Comment (hui444) [inline frontend/src/routes/Schedule/CreateEvent/index.tsx:54]: this requires `margin: 0 20px 0 0;`

**Review by hui444 — APPROVED** [review]
lgtm but please fix and resolve comments before merging!

File: frontend/src/components/Scheduling/ViewEventDetailCard.tsx (hunk 1 above)
Comment (jason-ng-zq99) [reply frontend/src/components/Scheduling/ViewEventDetailCard.tsx:69]: Resolved in 4f050bb

File: frontend/src/routes/FacilityBooking/CreateBooking/oldindex.tsx (hunk 2 above)
Comment (jason-ng-zq99) [reply frontend/src/routes/FacilityBooking/CreateBooking/oldindex.tsx:67]: Resolved in 4f050bb

File: frontend/src/routes/Schedule/CreateEvent/index.tsx (hunk 3 above)
Comment (jason-ng-zq99) [reply frontend/src/routes/Schedule/CreateEvent/index.tsx:54]: Resolved in 4f050bb

**Pull request merged**